django/contrib/admin/utils.py

```diff
@@ -521,3 +530,14 @@ def construct_change_message(form, formsets, add):
                         }
                     })
     return change_message
+
+
+def _get_changed_field_labels_from_form(form, changed_data):
+    changed_field_labels = []
+    for field_name in changed_data:
+        try:
+            verbose_field_name = form.fields[field_name].label or field_name
+        except KeyError:
+            verbose_field_name = field_name
+        changed_field_labels.append(str(verbose_field_name))
+    return changed_field_labels
```

Comment:
I would have liked to see that as part of the public API, unless you have an objection ?

Reply:
I'm not sure what you have in mind, but `django/contrib/admin/utils.py` doesn't contain any public APIs (and this code doesn't seem to be admin-specific). I think to make it a public API (e.g. in django.forms) we would want to see it used outside the admin.